SdlFreshstart/Sdl.Community.SdlFreshstart/ViewModel/StudioViewModel.cs

```diff
@@ -345,12 +328,7 @@ private void AddLocation(string path, string description, string alias)
 
 		private void GetInstalledVersions()
 		{
-			var installedVersions = _versionService.GetInstalledStudioVersions();
-			installedVersions.Sort((item1, item2) =>
-				item1.MajorVersion < item2.MajorVersion ? 1 :
-				item1.MajorVersion > item2.MajorVersion ? -1 : 0);
-
-			_studioVersionsCollection = new ObservableCollection<StudioVersion>(installedVersions);
+			_studioVersionsCollection = new ObservableCollection<StudioVersion>(_versionService.GetInstalledStudioVersions());
```

Comment:
I know this code was here before, please change the type of the collection we don't need it to be an observable collection because we don't remove or add any other version after we load the initial list.

Observable collection is not eficient like List.

Reply:
Done